spec/features/ncr_proposal_spec.rb

```diff
@@ -93,6 +92,18 @@
       expect(page).to have_content("RWA Number")
     end
 
+    it "hides fields based on expense", :js => true do
+      visit '/ncr/proposals/new'
+      expect(find_field("RWA Number", visible: false).visible?).to be false
+      expect(find_field("emergency", visible: false).visible?).to be false
+      choose 'BA61'
+      expect(find_field("RWA Number", visible: false).visible?).to be false
+      expect(find_field("emergency", visible: false).visible?).to be true
+      choose 'BA80'
+      expect(find_field("RWA Number", visible: false).visible?).to be true
+      expect(find_field("emergency", visible: false).visible?).to be false
```

Comment:
Minor: instead of being tied to the implementation (the fields being hidden rather than removed), thoughts about checking whether it's present at all or not?

```ruby
visit '/ncr/proposals/new'
expect(find_field("RWA Number")).to be_nil
expect(find_field("emergency")).to be_visible
# etc.
```